packages/cw-orch-mock/src/queriers/env.rs

```diff
@@ -9,10 +9,20 @@ impl<A: Api, S: StateInterface> EnvironmentQuerier for MockBase<A, S> {
     fn env_info(&self) -> EnvironmentInfo {
         let block_info = self.block_info().unwrap();
         let chain_id = block_info.chain_id.clone();
-        let chain_name = chain_id.rsplitn(2, '-').collect::<Vec<_>>()[1].to_string();
+        let chain_id_split = chain_id
+            .rsplitn(2, '-')
+            .map(ToString::to_string)
+            .collect::<Vec<_>>();
+
+        // We make sure this doesn't error even if the chain id doesn't have the `osmosis-1` format
+        let chain_name = if chain_id_split.len() > 1 {
+            &chain_id_split[1]
+        } else {
+            &chain_id_split[0]
+        };
```

Comment:
won't it be simpler to just chain_id.rsplitn(2, '-').last().unwrap().to_string()

Reply:
it won't panic on unwrap even with "" chain_id

Reply:
Oh yeah, I forgot it has a max of 2 elements